### PR TITLE
update samples operator to run e2e-aws-builds and e2e-aws-jenkins if …

### PR DESCRIPTION
**Pull request by gabemontero**
…we change the samples content

per recent experience with remove EOL images ... run both builds and jenkins e2e suites if we changes the actual set of samples installed by samples operator

/assign @adambkaplan 

@bparees @akram @waveywaves FYI

### CODE REVIEW

File: ci-operator/jobs/openshift/cluster-samples-operator/openshift-cluster-samples-operator-master-presubmits.yaml [hunk 1]
```diff
@@ -64,6 +64,70 @@ presubmits:
         secret:
           secretName: sentry-dsn
     trigger: (?m)^/test( | .* )e2e-aws,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```
Comment (bparees) [inline ci-operator/jobs/openshift/cluster-samples-operator/openshift-cluster-samples-operator-master-presubmits.yaml]: this should be false

File: ci-operator/jobs/openshift/cluster-samples-operator/openshift-cluster-samples-operator-master-presubmits.yaml [hunk 2]
```diff
@@ -128,6 +192,70 @@ presubmits:
         secret:
           secretName: sentry-dsn
     trigger: (?m)^/test( | .* )e2e-aws-image-ecosystem,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```
Comment (bparees) [inline ci-operator/jobs/openshift/cluster-samples-operator/openshift-cluster-samples-operator-master-presubmits.yaml]: should be false

**Comment by bparees** [comment]
not seeing the actual run_if_changed logic?

e.g. https://github.com/openshift/release/blob/master/ci-operator/jobs/openshift/origin/openshift-origin-master-presubmits.yaml#L11

**Comment by waveywaves** [comment]
ack

File: ci-operator/jobs/openshift/cluster-samples-operator/openshift-cluster-samples-operator-master-presubmits.yaml (hunk 1 above)
Comment (gabemontero) [reply ci-operator/jobs/openshift/cluster-samples-operator/openshift-cluster-samples-operator-master-presubmits.yaml]: yep part of next push

**Comment by gabemontero** [comment]
And yeah I swore I added `run_if_changed` .... too much multitasking again I suspect ... part of next push

**Comment by gabemontero** [comment]
updates pushed in separate commit @bparees .... will squash if you are good

**Comment by gabemontero** [comment]
Also e2e-aws-jenkins will fail until https://github.com/openshift/origin/pull/23481 merges

But @akram still has a couple of changes needed to be made in that PR

**Comment by bparees** [comment]
/lgtm
/hold

@gabemontero unhold when you're ready, not sure if you want this landed before the e2es are actually passing again.

**Comment by gabemontero** [comment]
the rehearsals won't pass @bparees, which I think will prevent merge, until https://github.com/openshift/origin/pull/23481lands

**Comment by gabemontero** [comment]
also, squashed commits, please reapply lgtm at your convenience @bparees , thanks

**Comment by bparees** [comment]
> the rehearsals won't pass @bparees, which I think will prevent merge,

ah, true, but it'll also keep retrying noisily without the hold.

/lgtm

**Comment by gabemontero** [comment]
/hold cancel

jenkins and builds e2e rehearsal passed @bparees

**Pull request merged**